tests/xharness/xharness.csproj

```diff
@@ -70,87 +70,78 @@
     <PackageReference Include="Mono.Options" Version="5.3.0.1" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="AppRunner.cs" />
+    <Compile Include="BCLTestImporter\BCLTestAssemblyDefinition.cs" />
+    <Compile Include="BCLTestImporter\BCLTestImportTargetFactory.cs" />
+    <Compile Include="BCLTestImporter\BCLTestInfoPlistGenerator.cs" />
+    <Compile Include="BCLTestImporter\BCLTestProjectDefinition.cs" />
+    <Compile Include="BCLTestImporter\BCLTestProjectGenerator.cs" />
+    <Compile Include="BCLTestImporter\Platform.cs" />
+    <Compile Include="BCLTestImporter\RegisterTypeGenerator.cs" />
+    <Compile Include="CrashReportSnapshot.cs" />
+    <Compile Include="DeviceLogCapturer.cs" />
+    <Compile Include="Execution\IProcessManager.cs" />
+    <Compile Include="Execution\ProcessManager.cs" />
+    <Compile Include="GitHub.cs" />
+    <Compile Include="Harness.cs" />
+    <Compile Include="Jenkins\Jenkins.cs" />
+    <Compile Include="Jenkins\Resource.cs" />
+    <Compile Include="Jenkins\Resources.cs" />
```

Comment:
Looks good. But what is going on with these lines.. I'm sure we already had the BCLTest* etc.. something being changed by the IDE?

Reply:
They are now sorted alphabetically. Should I try to revert this? Might be handy later on..

Reply:
Nah, not big problem. I was curious